code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -175,7 +175,10 @@
 		if(FEMALE)
 			t_him = "her"
 
-	if(target_zone in list("l_arm", "r_arm"))
+	if(target_zone == "head")
```

Comment:
thought - make it eyes (or generally something else) to avoid accidental triggering

Reply:
eyes are pretty annoying to select, and this would only trigger as much as the other emotes do, so i don't think this is an issue